client/signup/steps/import-from/wordpress/import-everything/confirm-upgrade-plan.tsx

```diff
@@ -0,0 +1,77 @@
+import { getPlan, PLAN_BUSINESS } from '@automattic/calypso-products';
+import { sprintf } from '@wordpress/i18n';
+import { check, Icon } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import QueryPlans from 'calypso/components/data/query-plans';
+import { getFeatureByKey } from 'calypso/lib/plans/features-list';
+import PlanPrice from 'calypso/my-sites/plan-price';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getPlanRawPrice } from 'calypso/state/plans/selectors';
+import { SitesItem } from 'calypso/state/selectors/get-sites-items';
+import type { FunctionComponent } from 'react';
+
+interface Props {
+	sourceSite: SitesItem | null;
+}
+
+export const ConfirmUpgradePlan: FunctionComponent< Props > = ( props ) => {
+	const { __ } = useI18n();
+
+	const { sourceSite } = props;
+	const plan = getPlan( PLAN_BUSINESS );
+	const planId = plan?.getProductId();
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	const promotedFeatures: string[] = plan?.getPromotedFeatures() ? plan?.getPromotedFeatures() : [];
```

Comment:
Maybe this can simply be `plan?.getPromotedFeatures() ?? []`.  Is `getPromotedFeatures` a future implementation?